### PR TITLE
add an app for vertical ngo set up

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: vertical_ngo/models/res_config.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        'Create tenders to negociate a framework agreements',
+        help="To allow your NGO to manage tenders when negociating "
+        "framework agreements from possible suppliers."
+    )
+    module_framework_agreement_sourcing = fields.Boolean(
+        'Allow to source a logistics requisition with a framework agreement',
+        help="if your NGO has framework agreements with suppliers "
+        "which you use to source logistics requisitions."
+    )
+    module_logistic_budget = fields.Boolean(
```
Comment (bealdav) [inline vertical_ngo/models/res_config.py:38]: Your module name(s) is 'logistic_budget'. It should be 'logistics_budget' ?
May be you could use https://github.com/bealdav/erp_rename/blob/master/erp_rename.py
I'm sure you can do a nice improvement of this basic tool and embedds it in your community maintener tools

Comment (gurneyalex) [inline vertical_ngo/models/res_config.py:38]: reported as #183

Thanks

**Comment by lepistone** [comment]
Thanks :+1:

File: README.md [hunk 2]
```diff
@@ -2,23 +2,39 @@
...
+This project aim to develop and publish all modules related to the need of
+humanitarian NGOs. It would cover the overall needs in terms of logistics,
+order management, accounting, transportation and distribution.
+
+Main changes are made in the procurement standard flow of Odoo. Usually, you
+have sales that drive procurement, that drive purchase. With those modules, you
+record logistic requisition to capture the needs, you source them from one way
+or another (purchase, stock, donnation,..) and give back to the requestor the
+price, time and product information. If he agree with, then you confirm the
+requisition by creating an offer. The offer will drive all the necessary flow
```
Comment (yvaucher) [inline README.md]: all the necessary flow**s**

File: README.md [hunk 3]
```diff
@@ -2,23 +2,39 @@
...
+to deliver the good where they are needed. It also improve the purchase
```
Comment (yvaucher) [inline README.md]: s/improve/improves

File: README.md [hunk 4]
```diff
@@ -2,23 +2,39 @@
...
-- NGO Addons: That contain the module specific to the NGO world (currently management of logistic reuquisition, kind of need registring and sourcing tools)
+- NGO Addons: That contain the module specific to the NGO world (currently
+  management of logistic reuquisition, kind of need registring and sourcing
+  tools)
 
 
-Other related community project where we put some useful module for NGO as well:
---------------------------------------------------------------------------------------------------------------------
+Other related community project where we put some useful module for NGO as well
+-------------------------------------------------------------------------------
```
Comment (yvaucher) [inline README.md:31]: stock-logistics-transportation to add ?

File: vertical_ngo/view/vertical_ngo.xml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+         <separator string="Shipment"/>
+          <group>
+            <label for="id" string="Features"/>
+            <div>
+              <div>
+                <field name="module_ngo_shipment_plan"
+                       class="oe_inline"/>
+                <label for="module_ngo_shipment_plan"/>
+              </div>
+<!--
```
Comment (yvaucher) [inline vertical_ngo/view/vertical_ngo.xml]: to remove ?

**Comment by yvaucher** [comment]
That's a great idea.

Few things to fix and  :+1:

File: vertical_ngo/view/vertical_ngo.xml (hunk 5 above)
Comment (lepistone) [inline vertical_ngo/view/vertical_ngo.xml]: @yvaucher should we remove that or replace it with another module?

**Comment by lepistone** [comment]
@yvaucher I addressed your remarks except the last one. Done that, I think we can merge.

File: vertical_ngo/view/vertical_ngo.xml [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+              <div>
+                <field name=""
+                       class="oe_inline"/>
+                <label for=""/>
+              </div>
+-->
```
Comment (yvaucher) [inline vertical_ngo/view/vertical_ngo.xml]: replying to https://github.com/OCA/vertical-ngo/pull/182/files#r32202746

I meant to remove this commented block I don't see the point of having this "template" here

**Pull request merged**